crates/shared/src/price_estimation/native_price_cache.rs

```diff
@@ -30,12 +30,12 @@ struct Inner {
 
 impl Inner {
     // Returns a single cached price and updates its `requested_at` field.
-    // If there is no entry for the token yet an outdated cache entry gets created.
     fn get_cached_price(
         token: &H160,
         now: Instant,
         cache: &mut MutexGuard<HashMap<H160, CachedPrice>>,
         max_age: &Duration,
+        create_missing_entry: bool,
```

Comment:
It's going to be unclear later why we have this. Could you add a comment somewhere explaining the same reasoning as in the PR.

Reply:
I already added it [here](https://github.com/cowprotocol/services/pull/1024/commits/50d6b585c6e24484b3259809441b38314dedb500#diff-50d12b254f11a969c7dc675e6f3181c39a3785e541dac0963f869d2d3ec9bacaR51-R52). Do you think it should rather be documented at the function signature?

Reply:
That's fine. In the diff I glanced over that this is extending the comment too.